x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/plan/physical/SequenceExec.java

```diff
@@ -29,27 +31,37 @@
     private final List<List<Attribute>> keys;
     private final Attribute timestamp;
     private final Attribute tiebreaker;
+    private final Limit limit;
+    private final OrderDirection direction;
 
     public SequenceExec(Source source,
                         List<List<Attribute>> keys,
                         List<PhysicalPlan> matches,
                         List<Attribute> untilKeys,
                         PhysicalPlan until,
                         Attribute timestamp,
-                        Attribute tiebreaker) {
-        this(source, combine(matches, until), combine(keys, singletonList(untilKeys)), timestamp, tiebreaker);
+                        Attribute tiebreaker, OrderDirection direction) {
```

Comment:
Minor: code formatting for consistency:
```suggestion
                        Attribute tiebreaker, 
                        OrderDirection direction) {
```